README.md

```diff
@@ -1,5 +1,4 @@
-[![Build](https://github.com/cesko-digital/loono-be/actions/workflows/master-check.yml/badge.svg)](https://github.com/cesko-digital/loono-be/actions/workflows/master-check.yml)
-[![Post deploy tests](https://github.com/cesko-digital/loono-be/actions/workflows/post-deploy-tests.yml/badge.svg)](https://github.com/cesko-digital/loono-be/actions/workflows/post-deploy-tests.yml)
+[![Build](https://github.com/cesko-digital/loono-be/actions/workflows/build_and_test_check.yml/badge.svg)](https://github.com/cesko-digital/loono-be/actions/workflows/master-check.yml)
```

Comment:
Mas tu odkaz na ten stary master-check.yml

Reply:
Jejda, dikes [1b0081f](https://github.com/cesko-digital/loono-be/pull/53/commits/1b0081fb37ed41d36d8304449b67cc4e1a9e6593)